.redhat-ci.yml

```diff
@@ -101,6 +108,7 @@ tests:
   - >
       docker run --privileged -v $(pwd):/srv/code --rm
       projectatomic/rpm-ostree-tester /bin/sh -c "
+      yum -y install  https://kojipkgs.fedoraproject.org//packages/ostree/2017.4/2.fc25/x86_64/ostree-2017.4-2.fc25.x86_64.rpm  https://kojipkgs.fedoraproject.org//packages/ostree/2017.4/2.fc25/x86_64/ostree-devel-2017.4-2.fc25.x86_64.rpm https://kojipkgs.fedoraproject.org//packages/ostree/2017.4/2.fc25/x86_64/ostree-libs-2017.4-2.fc25.x86_64.rpm &&
```

Comment:
You should be able to split this on multiple lines to make it more legible, even with indentation if you'd like.